src/OpenColorIO/fileformats/ctf/CTFReaderHelper.cpp

```diff
@@ -2564,39 +2564,40 @@ void CTFReaderLogElt_2_0::end()
 {
     CTFReaderOpElt::end();
 
-    const auto dir = LogUtil::GetLogDirection(m_ctfParams.m_style);
-    m_log->setDirection(dir);
+    const auto dir = LogUtil::GetLogDirection(getCTFParams().m_style);
+    getLog()->setDirection(dir);
 
-    if (m_ctfParams.getType() == LogUtil::CTFParams::CINEON)
+    if (getCTFParams().getType() == LogUtil::CTFParams::CINEON)
     {
         double base = 2.0;
         LogOpData::Params rParams, gParams, bParams;
 
         try
         {
             // This handles all log styles.
-            LogUtil::ConvertLogParameters(m_ctfParams, base, rParams, gParams, bParams);
+            LogUtil::ConvertLogParameters(getCTFParams(), base, rParams, gParams, bParams);
         }
         catch (Exception& ce)
         {
             ThrowM(*this, "Parameters are not valid: '", ce.what(), "'. ");
         }
 
-        m_log->setBase(base);
-        m_log->setRedParams(rParams);
-        m_log->setGreenParams(gParams);
-        m_log->setBlueParams(bParams);
+        getLog()->setBase(base);
+        getLog()->setRedParams(rParams);
+        getLog()->setGreenParams(gParams);
+        getLog()->setBlueParams(bParams);
     }
     else
     {
-        if (!m_baseSet)
+        if (!isBaseSet())
         {
-            if (m_ctfParams.m_style == LogUtil::LOG2 || m_ctfParams.m_style == LogUtil::ANTI_LOG2)
+            if (getCTFParams().m_style == LogUtil::LOG2 
+                    || getCTFParams().m_style == LogUtil::ANTI_LOG2)
```

Comment:
[minor] indentation alignment

Reply:
That's how we put multi-line conditions everywhere in the code.